### PR TITLE
[DCOS-38717] Add docs for confluent zookeeper 2.3.0-4.0.0e release

### PR DESCRIPTION
**Pull request by elezar**
## Description
https://jira.mesosphere.com/browse/DCOS-38717

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+excerpt:
+title: Release Notes
+menuWeight: 120
+model: /services/kafka-zookeeper/data.yml
+render: mustache
+---
+# Version 2.3.0-4.0.0e
+
+## Features
+- All frameworks (Confluent ZooKeeper included) now isolate their `/tmp` task directories by making them Mesos [`SANDBOX_PATH` volume sources](https://github.com/apache/mesos/blob/master/docs/container-volume.md#sandbox_path-volume-source). ([#2467](https://github.com/mesosphere/dcos-commons/pull/2467) and [#2486](https://github.com/mesosphere/dcos-commons/pull/2486))
```
Comment (mpereira) [inline pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md]: `s/Confluent ZooKeeper/{{ model.techName }}/`?

File: pages/services/confluent-zookeeper/2.3.0-4.0.0e/security/index.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+example/zookeeper-1-server.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+example/zookeeper-2-server.agoodexample.autoip.dcos.thisdcos.directory@EXAMPLE
+```
+#include /services/include/security-kerberos-ad.tmpl
+
+#include /services/include/security-service-keytab.tmpl
+
+#### Install the Service
+
+Install the DC/OS {{ model.techName }} service withe the following options in addition to your own:
```
Comment (mpereira) [inline pages/services/confluent-zookeeper/2.3.0-4.0.0e/security/index.md]: `s/withe/with`

File: pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md (hunk 1 above)
Comment (elezar) [reply pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md]: Then I have to edit it again when I copy this text to the github release notes 😢

Comment (mpereira) [reply pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md]: Ah, it's fine then.

**Comment by elezar** [comment]
@SarahStegall I have tried to incorporate your security template changes into the new Confluent Kafka docs. Let me know if I'm missing anything.

**Pull request merged**